src/pages/index.js

```diff
@@ -0,0 +1,9 @@
+import HomePage from './homePage/HomePage'
+import AboutUs from './aboutUs/AboutUs'
+import ContactUs from './contactUs/ContactUs'
+import JoinUs from './joinUs/JoinUs'
+import Members from './members/Members'
+import Login from './login/login'
+import NotFound from './notFound/NotFound'
+
```

Comment:
use Navbar here as:
import {Navbar} from './globalComponents'

and then export it.